src/modules/commander/Commander.cpp

```diff
@@ -2277,9 +2278,9 @@ Commander::run()
 		}
 
 		if (armed.armed &&
-			failure_detector_updated &&
-			!_in_flight_termination &&
-			!status_flags.circuit_breaker_flight_termination_disabled) {
+		    failure_detector_updated &&
+		    !_in_flight_termination &&
+		    !status_flags.circuit_breaker_flight_termination_disabled) {
```

Comment:
Next time try to make a separate commit for formatting only.

Reply:
Wow so many negations :smile:, let me try wrap my head around this:

So if the circuit breaker is enabled which is the default then the variable `circuit_breaker_flight_termination_disabled` is `true` meaning the flight termination is "disabled", so nothing is happening. Now if the circuit breaker is disabled (set to 0), the flight termination is "enabled", so "no disabled", and therefore this block is called.

I think that's correct.